test/main_test.go

```diff
@@ -168,7 +168,7 @@ func getFilesMetadatasWithContent(t testing.TB, filePath string, content []byte)
 	files := make(model.FileMetadatas, 0)
 
 	for _, parser := range combinedParser {
-		docs, err := parser.Parse(filePath, content)
+		docs, err := parser.Parse(filePath, content, true)
```

Comment:
same here on the default value

Reply:
@pereiramarco011 we need to pass as true, because this test runs queries related with openapi